design-document.md

```diff
@@ -115,6 +293,50 @@ collude and break soundness. Is this a contingency we need to address? There are
 techniques in [BBC+19] that account for this; we need to figure out if they're
 practical.]]
 
+### Future work and possible extensions
+
+In this section we discuss attacks that are not considered in the above threat
+model, and suggest mitigations that could be incorporated into implementations
+of this protocol or future revisions of this specfication.
+
+#### Client authentication
+
+Attackers can impersonate Prio clients and submit large amounts of valid but
+false data in order to spoil aggregations. Deployments could require that
+clients present a trusted identity before they may contribute data. For example,
+by requiring data submissions to be signed with a key trusted by aggregators. A
+variety of techniques providing different degrees of assurance could be employed
+to establish trust in clients, ranging from whitebox cryptography to public key
+infrastructures. However some deployments may opt to accept the risk of false
+data to avoid having to figure out how to distribute trusted identities to
+clients.
+
+#### Client attestation
+
+In the current threat model, servers participating in the protocol have no
+insight into the activities of clients except that they have uploaded data into
+a Prio aggregation, meaning that clients could covertly leak a user's data into
+some other channel which compromises privacy and anonymity. If we introduce the
```

Comment:
```suggestion
some other channel which compromises privacy. If we introduce the
```

As I mentioned above, privacy implies anonymity as the terms are defined above. (You may disagree?)